Assets/Scripts/UnitData.cs

```diff
@@ -4,6 +4,9 @@
 public class UnitData : ScriptableObject {
     public string unitName;
 
-    public int healthPoints;
-    public int damage;
+    public int movementSpeed = 50;
+    public int healthPoints = 100;
+    public int damage = 10;
+    public int shootingRange = 3;
+    public GameEnums.EntityType entityType = GameEnums.EntityType.CONTACT;
```

Comment:
In continuation of the arrow adjustments I wrote about before, I suggest we add a field here where we can set the projectile prefab to then use when spawning them later.